src/SearchResults.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResult = props => {
+  const rows = props.results.map((user, index) => (
+    <tr key={index}>
+      <th scope="row">{user.id}</th>
+      <td>{user.title}</td>
+      <td>{user.firstName}</td>
+      <td>{user.surname}</td>
+      <td>{user.email}</td>
+      <td>{user.roomId}</td>
+      <td>{user.checkInDate}</td>
+      <td>{user.checkOutDate}</td>
+      <td>{moment(user.checkOutDate).diff(user.checkInDate, "days")} days</td>
+    </tr>
+  ));
+  return (
+    <div class="table-responsive">
+      <table className="table">
+        <thead className="thead-dark">
+          <tr>
+            <th scope="col">id</th>
+            <th scope="col">title</th>
+            <th scope="col">first name</th>
+            <th scope="col">surname</th>
+            <th scope="col">email</th>
+            <th scope="col">room id </th>
+            <th scope="col">check in date</th>
+            <th scope="col">check out data</th>
+            <th scope="col">number of nights</th>
+          </tr>
+        </thead>
+        <tbody className="result">{rows}</tbody>
```

Comment:
Similar to above, this could be "inlined" by inserting the code assigned to `rows` here.

But this is fine too! And sometimes I prefer this approach 🙂

Reply:
Thank you very much for that and I did come across I can use "inlined" when I started working on High-Sores-Table project. :)